### DISCUSSION TITLE
Update to rely on Illuminate Collections instead of Tightenco/Collect

### DISCUSSION THREAD

**Discussion by mattstauffer** [laravel/valet#1137] [Ideas]
Any deprecation issues we need to consider? It'll force everyone to update their installers but they'll have to anyway. Any other global packages that would stop being compatible to be installed at the same time if we did this?

**Comment by soilSpoon** [comment]
`illuminate/collections` exists since `^8.0`. It only supports `PHP` `^7.3` and `^8.0`.
But valet supports `PHP` `^5.6` and above.

**Comment by wturrell** [comment]
FYI, Valet is now impacted by this: https://github.com/tighten/collect/issues/261 with PHP 8.1

**Comment by driesvints** [comment]
Started working on this here: https://github.com/laravel/valet/pull/1114
